runtime/build.rs

```diff
@@ -0,0 +1,58 @@
+static NETWORKS: &[(&str, &[&str])] = &[
```

Comment:
We will also have to remove these, they are builtin-actors-specific and I don't think we'll need them.

Reply:
removed